### PR TITLE
Ria 3378 ho auth

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3378

### Change description ###
Added response logging for testing and more functional tests with PRP1 data

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-3378-expected-400-response-invalid-format.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "submitAppeal",
+      "state": "appealSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "homeOfficeReferenceNumber": "1212/009900362015"
```
Comment (raghera) [inline src/functionalTest/resources/scenarios/RIA-3378-expected-400-response-invalid-format.json:12]: Are these the home office provided id's?

Comment (jkumar-moj) [reply src/functionalTest/resources/scenarios/RIA-3378-expected-400-response-invalid-format.json:12]: That's correct @raghera . All of these IDs are based on the different scenarios to test against PRP1

**Review by divisathyan — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**